### PR TITLE
Laserjet printers

### PR DESCRIPTION
**Pull request by creviera**
## Description

Add support for laserjet printers to the export script.

Tested with laserjet m404n.

### CODE REVIEW

**Comment by emkll** [comment]
@creviera mind rebasing this PR on latest `master`? we should make use of the test improvements introduced in https://github.com/freedomofpress/securedrop-export/pull/31

**Comment by creviera** [comment]
So at the time I wrote this, we were building and testing for stretch. But now that securedrop-workstation has upgraded to buster, I need to run through testing this again.

**Comment by creviera** [comment]
I tested your commit with the `lpadmin` -> `cupsaccept` and was able to successfully print a document from the client.

**Comment by creviera** [comment]
Followup: https://github.com/freedomofpress/securedrop-export/issues/37

I've tested printing docx, doc, pdf, and txt on laserjet m404n

File: securedrop_export/export.py [hunk 1]
```diff
@@ -359,19 +361,26 @@ def get_printer_uri(self):
         return printer_uri
 
     def install_printer_ppd(self, uri):
+        if "Brother" in uri:
+            self.printer_driver = BRLASER_DRIVER
+            self.printer_ppd = BRLASER_PPD
+        elif "LaserJet" in uri:
+            self.printer_driver = LASERJET_DRIVER
+            self.printer_ppd = LASERJET_PPD
```
Comment (redshiftzero) [inline securedrop_export/export.py]: with this change, I can confirm that printing using your branch in securedrop-client and this version of securedrop-export all works without issue

**Comment by emkll** [comment]
I just re-tested these changes using a Brother printer, works as expected

**Comment by creviera** [comment]
Pushed unit tests and removed `printer_driver` and `printer_ppd` from the submission class (as discussed with @redshiftzero). This is in a final state.

File: tests/test_export.py [hunk 2]
```diff
@@ -315,3 +323,52 @@ def test_safe_check_call(capsys, mocker):
...
+
+def test_install_printer_ppd_error_no_driver(mocker):
+    submission = export.SDExport("testfile", TEST_CONFIG)
+    mocked_exit = mocker.patch.object(submission, "exit_gracefully", return_value=0)
+    mocker.patch("subprocess.check_call", side_effect=CalledProcessError(1, 'check_call'))
+
+    submission.install_printer_ppd("usb://HP/LaserJet%20Pro%20M404-M405?serial=A00000A000000")
+
+    assert mocked_exit.mock_calls[0][2]['msg'] == "ERROR_PRINTER_DRIVER_UNAVAILABLE"
+    assert mocked_exit.mock_calls[0][2]['e'] is None
```
Comment (redshiftzero) [inline tests/test_export.py]: thanks for the nice unit tests!

**Pull request merged**